torch/nn/qat/modules/activations.py

```diff
@@ -41,3 +41,34 @@ def from_float(cls, mod, qconfig=None):
         qconfig = mod.qconfig
         qat_hardswish = cls(qconfig=qconfig)
         return qat_hardswish
+
+class ELU(nn.ELU):
+    r"""This is the QAT equivalent of `torch.nn.ELU`.
+    """
+    _FLOAT_MODULE = nn.ELU
+
+    def __init__(self, alpha, qconfig=None):
+        super(ELU, self).__init__(alpha)
+        assert qconfig, 'qconfig must be provided for QAT module'
+        self.qconfig = qconfig
+        self.activation_post_process = qconfig.activation()
+
+    def forward(self, input):
+        return self.activation_post_process(F.elu(input, self.alpha))
+
+    @classmethod
+    def from_float(cls, mod, qconfig=None):
+        r"""Create a qat module from a float module or qparams_dict
+
+            Args: `mod` a float module, either produced by torch.quantization utilities
+            or directly from user
+        """
+        assert type(mod) == cls._FLOAT_MODULE, ' qat.' + cls.__name__ + '.from_float only works for ' + \
```

Comment:
btw we can probably have a helper function to assert this

Reply:
makes sense.  Might be good to do in a separate PR with just that change, since currently every eager mode module has this code snippet.

Reply:
yeah, sounds good